packages/components-js/tests/e2e/puppeteer/specs/toast.e2e.ts

```diff
@@ -30,9 +30,9 @@ const initToast = (opts?: InitToastOptions): Promise<void> => {
   };
 
   const style = `<style>p-toast {
-  --p-override-toast-timeout: ${TOAST_TIMEOUT_DURATION_OVERRIDE};
-  --p-override-toast-skip-timeout: false;
-  ${withAnimation ? `--p-override-toast-animation-duration: ${ANIMATION_DURATION / 1000}s` : ''}
+  --p-temporary-toast-override-timeout: ${TOAST_TIMEOUT_DURATION_OVERRIDE};
```

Comment:
Why do we need `override` in naming of those `temporary` css variables?

Reply:
There are two variables here, `--p-temporary-toast-override-timeout` & `--p-temporary-toast-skip-timeout`. I found it more clear if we leave it with override to distinguish between those two.

Reply:
I think `--p-temporary-toast-timeout` and `--p-temporary-toast-skip-timeout` are just fine. Because first accepts miliseconds and the second a boolean, right? Actually not sure why we need both, but there is a reason for sure. ;-)

Reply:
It is needed for different test scenarios in e2e test.